x/evm/keeper/msg_server_test.go

```diff
@@ -126,7 +126,7 @@ func TestCreateApproveContractCalls(t *testing.T) {
 						PayloadHash:      types.Hash(evmCrypto.Keccak256Hash(payload)),
 						Payload:          payload,
 						Symbol:           rand.Denom(3, 5),
-						Amount:           sdk.NewUint(uint64(rand.I64Between(1, 10000))),
+						Amount:           sdk.NewUint(uint64(rand.I64Between(2, 10000))),
```

Comment:
later in the test there is a `rand.I64Between(1, event.Amount)`. If the amount starts at 1 there is a possibility the test fails (I rand into that case)